crates/cargo-util-schemas/src/manifest/mod.rs

```diff
@@ -35,11 +35,13 @@ use crate::schema::TomlValueWrapper;
 #[serde(rename_all = "kebab-case")]
 #[cfg_attr(feature = "unstable-schema", derive(schemars::JsonSchema))]
 pub struct TomlManifest {
-    // when adding new fields, be sure to check whether `requires_package` should disallow them
     pub cargo_features: Option<Vec<String>>,
+
+    // Update `requires_package` when adding new package-specific fields
     pub package: Option<Box<TomlPackage>>,
     pub project: Option<Box<TomlPackage>>,
-    pub profile: Option<TomlProfiles>,
+    pub badges: Option<BTreeMap<String, BTreeMap<String, String>>>,
```

Comment:
I wish we could have a more automated way to group these. Having a separate strict is too overkill. Anyway reorder looks fine!